bucket/teraterm.json

```diff
@@ -0,0 +1,25 @@
+{
+    "homepage": "https://osdn.net/projects/ttssh2/",
+    "description": "Terminal emulator for ssh/telnet/serial connection.",
+    "version": "4.103",
+    "license": "BSD-3-Clause",
+    "url": "http://pumath.dl.osdn.jp/ttssh2/71232/teraterm-4.103.zip",
+    "hash": "99c9b7559c0551412ed75673f2449d84f25a945e351c7a76689f5751628d56d5",
+    "extract_dir": "teraterm-4.103",
+    "bin": "ttermpro.exe",
+    "shortcuts": [
+        [
+            "ttermpro.exe",
+            "Tera Term"
+        ]
+    ],
+    "checkver": "releases/(?<tag>\\d+)\">Tera Term ([\\d.]+)</a>",
+    "autoupdate": {
+        "url": "http://pumath.dl.osdn.jp/ttssh2/$matchTag/teraterm-$version.zip",
+        "extract_dir": "teraterm-$version",
+        "hash": {
+            "url": "https://osdn.net/projects/ttssh2/downloads/$matchTag/teraterm-$version.zip/",
+            "regex": "SHA256</dt>[\\n\\s]+<dd>([0-9a-f]+)</dd>"
```

Comment:
Test your autoudpates before posting PR. This obviously do not work without mulitline

```suggestion
            "regex": "(?sm)MD5</dt>[\\n\\s]+<dd>$md5</dd>"
```

Reply:
It works well for me... Why not use sha256? It's safer.

```
Searching hash for teraterm-4.103.zip in https://osdn.net/dl/ttssh2/teraterm-4.103.zip
Found: 99c9b7559c0551412ed75673f2449d84f25a945e351c7a76689f5751628d56d5 using Extract Mode
```

Reply:
Thanks for the suggestion.

I did tested the *autoupdate*. (and it's working, is `multiline` enabled by default?)
![hash](https://user-images.githubusercontent.com/27724471/60753967-db408080-a00c-11e9-9432-0977fc59c67d.png)